src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="main">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--2">
+        <div class="stars__star active"></div>
+        <div class="stars__star active"></div>
```

Comment:
```suggestion
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```
Consider using class with a modifier to set the number of active stars